components/jsonSchemaLinks.ts

```diff
@@ -1,36 +1,40 @@
-    'enum': '/understanding-json-schema/reference/generic#enum',
-    'title': '/understanding-json-schema/reference/generic#annotation',
-    'description': '/understanding-json-schema/reference/generic#annotation',
-    'default': '/understanding-json-schema/reference/generic#annotation',
-    '$id': '/understanding-json-schema/basics#declaring-a-unique-identifier',
-    '$schema': '/understanding-json-schema/basics#declaring-a-json-schema',
-    'multipleOf': '/understanding-json-schema/reference/numeric#multiples',
-    'required': '/understanding-json-schema/reference/object#required',
-    'patternProperties': '/understanding-json-schema/reference/object#patternProperties',
-    'additionalProperties': '/understanding-json-schema/reference/object#additionalproperties',
-    'dependentRequired': '/understanding-json-schema/reference/conditionals#dependentRequired',
-    'dependentSchemas': '/understanding-json-schema/reference/conditionals#dependentSchemas',
-    'items': '/understanding-json-schema/reference/array#items',
-    '$ref': '/understanding-json-schema/structuring#dollarref'
+    type: '/understanding-json-schema/reference/type',
+    properties: '/understanding-json-schema/reference/object#properties',
+    pattern: '/understanding-json-schema/reference/string#regexp',
+    minimum: '/understanding-json-schema/reference/numeric#range',
+    exclusiveMinimum: '/understanding-json-schema/reference/numeric#range',
+    maximum: '/understanding-json-schema/reference/numeric#range',
+    exclusiveMaximum: '/understanding-json-schema/reference/numeric#range',
+    minLength: '/understanding-json-schema/reference/string#length',
+    maxLength: '/understanding-json-schema/reference/string#length',
+    enum: '/understanding-json-schema/reference/generic#enum',
+    title: '/understanding-json-schema/reference/generic#annotation',
+    description: '/understanding-json-schema/reference/generic#annotation',
+    default: '/understanding-json-schema/reference/generic#annotation',
+    $id: '/understanding-json-schema/basics#declaring-a-unique-identifier',
+    $schema: '/understanding-json-schema/basics#declaring-a-json-schema',
+    multipleOf: '/understanding-json-schema/reference/numeric#multiples',
+    required: '/understanding-json-schema/reference/object#required',
+    patternProperties:
+      '/understanding-json-schema/reference/object#patternProperties',
+    additionalProperties:
+      '/understanding-json-schema/reference/object#additionalproperties',
+    dependentRequired:
+      '/understanding-json-schema/reference/conditionals#dependentRequired',
+    dependentSchemas:
+      '/understanding-json-schema/reference/conditionals#dependentSchemas',
+    items: '/understanding-json-schema/reference/array#items',
+    $ref: '/understanding-json-schema/structuring#dollarref',
   },
   stringValue: {
     '"null"': '/understanding-json-schema/reference/null',
```

Comment:
@benjagm  @Relequestual , I saw small bug in the code. I think there should be no quotes in object key. Prettier Formatting might cause some bugs.

Reply:
Not sure if that is a bug. @musemind do you remember if this code is correct this way?

Reply:
@benjagm I went through the codebase and found that this code is already present as it is. It has not been formatted by Prettier and Prettier is working fine with it.

Thank you : )